src/clojure/puppetlabs/ssl_utils/core.clj

```diff
@@ -1043,3 +1042,19 @@
   [c :- CertOrCSR
    digest :- schema/Str]
   (SSLUtils/getFingerprint c digest))
+
+(schema/defn ^:always-validate prevent-duplicate :- (schema/maybe '(schema/Int))
```

Comment:
I think this function name could be a bit clearer - perhaps something like, `remove-already-revoked-serials`?